pkgs/development/libraries/science/math/libtorch/binary-hashes.nix

```
@@ -1,19 +1,19 @@
-version : builtins.getAttr version {
-  "2.0.0" = {
+version: builtins.getAttr version {
+ "2.2.2" = {
```

```suggestion
 "2.2.2" = {
     aarch64-darwin-cpu = {
      name = "libtorch-macos-arm64-2.2.2.zip";
      url = "https://download.pytorch.org/libtorch/cpu/libtorch-macos-arm64-2.2.2.zip";
      hash = "sha256-zXviMxGW0wpJDIbb++qaJLL8OwJPuT38KqRRXkEFGeA=";
    };
```

Hello @pinpox! `aarch64` support snippet ^